Doc/whatsnew/3.14.rst

```diff
@@ -786,6 +786,14 @@ linecache
   (Contributed by Tian Gao in :gh:`131638`.)
 
 
+logging.handlers
+----------------
+
+* :class:`logging.handlers.QueueListener` now implements the context
```

Comment:
Maybe mention that `RuntimeError` is not raised when the listener is started multiple times.